pkg/crc/machine/bundle/repository.go

```diff
@@ -45,13 +41,13 @@ type Repository struct {
 }
 
 func (repo *Repository) Get(bundleName string) (*CrcBundleInfo, error) {
-	var bundleInfo CrcBundleInfo
-	bundleInfo.cachedPath = getCachedBundlePath(repo.CacheDir, bundleName)
-	if !bundleInfo.isCached() {
-		return nil, fmt.Errorf("could not find cached bundle info in %s", bundleInfo.cachedPath)
+	path := getCachedBundlePath(repo.CacheDir, bundleName)
+	if _, err := os.Stat(path); err != nil {
+		return nil, errors.Wrapf(err, "could not find cached bundle info in %s", path)
```

Comment:
I'd prefer we stop using `errors.Wrapf` in new code as I don't think this integrates well with https://blog.golang.org/go1.13-errors (or am I wrong?)

Reply:
good to know! I didnt know. 

Apparently pkg/errors does the job: https://github.com/pkg/errors/blob/master/errors.go#L162

Reply:
Ah indeed, got misled by the end of their README https://github.com/pkg/errors/blob/master/README.md#roadmap